### PR TITLE
Added a deluxe cheeseburger

### PR DESCRIPTION
**Pull request by Narwhal8**
#### Summary
 None

#### Purpose of change

One game when I was playing I came across all of the stuff to make a good cheeseburger with lettuce, tomato and everything else that makes up a classic cheeseburger, but when I went to go and craft it I saw that this option did not exist! The only option was the classic cheeseburger which had only the option for 1 extra condiment and vegetable, so I figured that I might as well add in a deluxe form of the cheeseburger which has all of these fixings as a option for players while leaving in the classic burger. In this change I also added the deluxe cheeseburger to loot tables and slightly reduced the likelihood of finding a normal cheeseburger to compensate for this.

#### Describe the solution

Well its just a fancy cheeseburger, you craft it with bread, cheese, meat, pickle and/or picked veggies, onion, tomato, ketchup, mustard and mayo/horseradish

#### Describe alternatives you've considered

I also considered changing the original cheeseburger, but they seem nice for earlier play when you don't have a wide variety of vegetables to make food with and the normal cheeseburger is fine as is

#### Testing

I checked in game to see the stats that it would have on average compared to the normal crafted burger and it seemed a bit better without anything looking too overpowered as well as made sure there was nothing strange with it like indefinate shelf life

#### Additional context

None

### CODE REVIEW

**Comment by actual-nh** [comment]
To pass the (required) "Validate PR" test, you'll need to change your summary from `Category "Brief description"` to something like just `None`. (The formatting is checked because summaries - other than None - are used to create the first draft of the changelog for each release.)

**Comment by Narwhal8** [comment]
> To pass the (required) "Validate PR" test, you'll need to change your summary from `Category "Brief description"` to something like just `None`. (The formatting is checked because summaries - other than None - are used to create the first draft of the changelog for each release.)

Thanks! I had it edited before but I realized I deleted text I wasn't supposed to so I copy pasted it over to another PR, but seems like I forgot it

File: data/json/items/comestibles/meat_dishes.json [hunk 1]
```diff
@@ -1107,6 +1107,21 @@
...
     "vitamins": [ [ "vitA", 17 ], [ "vitC", 1 ], [ "calcium", 57 ], [ "iron", 65 ], [ "vitB", 213 ] ]
   },
+  {
+    "type": "COMESTIBLE",
+    "id": "deluxe_cheeseburger",
+    "copy-from": "cheeseburger",
+    "name": { "str": "deluxe cheeseburger" },
+    "weight": "380 g",
+    "calories": 500,
+    "description": "A cheeseburgur with more vegetables and condiments, This is the apex of pre-Cataclysm culinary achievement.",
```
Comment (Maleclypse) [inline data/json/items/comestibles/meat_dishes.json:1117]:
```suggestion
    "description": "A cheeseburger with more vegetables and condiments, This is the apex of pre-Cataclysm culinary achievement.",
```

File: data/json/items/comestibles/meat_dishes.json [hunk 2]
```diff
@@ -1097,7 +1097,7 @@
     "comestible_type": "FOOD",
     "symbol": "%",
     "calories": 380,
-    "description": "A sandwich of minced meat and cheese with condiments.  The apex of pre-Cataclysm culinary achievement.",
+    "description": "A sandwich of minced meat and cheese with condiments. A simple favorite of the people before the cataclysm.",
```
Comment (casswedson) [inline data/json/items/comestibles/meat_dishes.json:1100]:
```suggestion
    "description": "A sandwich of minced meat and cheese with condiments.  A simple favorite of the people before the cataclysm.",
```

File: data/json/items/comestibles/meat_dishes.json [hunk 3]
```diff
@@ -1107,6 +1107,21 @@
...
     "vitamins": [ [ "vitA", 17 ], [ "vitC", 1 ], [ "calcium", 57 ], [ "iron", 65 ], [ "vitB", 213 ] ]
   },
+  {
+    "type": "COMESTIBLE",
+    "id": "deluxe_cheeseburger",
+    "copy-from": "cheeseburger",
+    "name": { "str": "deluxe cheeseburger" },
+    "weight": "380 g",
+    "calories": 500,
+    "description": "A cheeseburger with more vegetables and condiments, This is the apex of pre-Cataclysm culinary achievement.",
```
Comment (casswedson) [inline data/json/items/comestibles/meat_dishes.json:1117]:
```suggestion
    "description": "A cheeseburger with more vegetables and condiments.  This is the apex of pre-Cataclysm culinary achievement.",
```

**Pull request merged**